modules/page_base.py

```diff
@@ -64,6 +64,20 @@ def __init__(self, driver, **kwargs):
                 manifest_name += f"_{char.lower()}"
         self.load_element_manifest(f"./modules/data/{manifest_name}.components.json")
 
+    _xul_source_snippet = (
+        'xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"'
+    )
+
+    def set_chrome_context(self):
+        """Make sure the Selenium driver is using CONTEXT_CHROME"""
+        if self._xul_source_snippet not in self.driver.page_source:
+            self.driver.set_context(self.driver.CONTEXT_CHROME)
+
```

Comment:
Would be cool to make this a context manager so you don't have to set `content` context.

Reply:
Yeah, the context manager is preferable, but there are places where I can't rely on it. I did go through and CM-ify any of the code where it wouldn't break the test.